content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,126 @@
 Italy
 =====
 
-Allow the user to generate the EDI document for Italian invoicing.
+Configuration
+=============
+The first configuration to book accounting transactions on an Italian localization is the
+:guilabel:`Italy - Accounting` module. It allows to:
 
-This module allows the creation of the EDI documents and the communication with the SDICoop web
-service of FatturaPA to send or receive invoices and notifications.
+- Generate electronic invoices in **FatturaPA's** format.
+- Send and receive electronic invoicing through SDIcoop.
 
 .. warning::
    Once this module is installed, it's no longer possible to send invoices via :ref:`PEC mails
    <italy/pec>`.
 
-SDICoop
-=======
+Once this module is installed, there are a few more necessary configurations. For all B2B
+transactions, Italy requires electronic invoicing with the use of specific formats and the
+transmission of invoices on **SDI** platform.
 
-Configuration
--------------
+Company information
+-------------------
+
+The company information is the also an important configuration to ensure the proper set up of your
+Accounting database. To add the information, go to :menuselection:`Configuration --> Settings -->
+General Settings --> Update info`. In this menu, make sure to have at least the following:
+
+- the company address
+- :guilabel:`GSTIN` code
+- :guilabel:`Codice Fiscale`
+- :guilabel:`Tax System`
+- :guilabel:`PEC address email`
 
-Setup the Codice Destinatario on FatturaPA
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+PEC
+---
+The PEC email is a specific type of certified email providing a legal equivalent to the traditional
+registered mail. The PEC email of the main company must be the same as the one registered by the
+Agenzia delle Entrate authorities.
+
+SDICoop
+-------
 
 To receive invoices and notifications from third parties, you need to inform the FatturaPA service
```

Comment:
I'd change the formulation a bit, I give you some suggestion you're free to use as a guidance
I would name the section "E-invoicing", briefly describe what the SdI (Exchange System) is, and then how to subscribe to it so that Odoo can receive the invoices from it (the section you already wrote below here).
The SdI is the public technical system that allows the sending and receiving of invoices between businesses, that allows the e-invoices in XML to be validated and sent to the other party, and that this process is mandatory by law. (Very) Basic reference in ENG here: https://www.fatturapa.gov.it/en/lafatturapa/